### ISSUE TITLE
 PMs Unreadable, vB3.7 --> MyBB1.6

### ISSUE THREAD

**Issue by JovanJ** [mybb/merge-system#24]
Description: When merging from vBulletin 3.7 to MyBB 1.6, all private messages became unreadable. Moreover, they all appeared in the "Sent Items" folder with no title, sent to "N/A (not sent)". See image below.

Forum System: vBulletin 3.7 to MyBB 1.6

http://dev.mybb.com/issues/1523

**Comment by LakesGeek** [comment]
Same on vB 3.8.  To fix it, I edited /merge/boards/vbulletin3/privatemessages.php

Find the section

```
if($data['folderid'] == -1)
        {
            $insert_data['folder'] = 2;
        }
        else
        {
            $insert_data['folder'] = 0;
        }

        $insert_data['subject'] = encode_to_utf8($data['subject'], "pm", "privatemessages");
```

Replace with

```
if($data['folderid'] == -1)
        {
            $insert_data['folder'] = 2;
        }
        else
        {
            $insert_data['folder'] = 1;
        }

        $insert_data['subject'] = encode_to_utf8($data['title'], "pmtext", "privatemessages");
```

Truncate the privatemessages table, restart the import, and just do the users (it ignores the existing ones, I noticed, so it's okay) and PMs again, then clean up.

**Comment by JN-Jones** [comment]
Closing this as we don't support vB 3 anymore.

**Comment by JN-Jones** [comment]
Reopening as we readded vB3

**Comment by JN-Jones** [comment]
This has been fixed with #97.
